### PR TITLE
Hand menu upgrades

### PR DESCRIPTION
**Pull request by rickfromwork**
Notable changes:
- Hand menu follows the left hand now by default, as most users are right-handed and this allows them to press buttons using their right hand
- Introduced visibility options, similar to MRTK's settings - The user should be looking at their hand, and the palm should be facing the user
- Moved the default position to the outer hand to make it easier to use when the palm is up (no need to overlap hands to interact with it)
- Near menu pin has been re-aligned after previous alignment changed to the TouchHolographicButton
- Padding on menus has been reduced
- Menu button scaling has been generalized to base TouchHolographicMenu class
- Fixed a failure that occurred when a menu tried updating its backplate but had no children

Done as part of #10846

### CODE REVIEW

File: gui/src/3D/controls/touchHolographicMenu.ts [hunk 1]
```diff
@@ -38,11 +43,13 @@ export class TouchHolographicMenu extends VolumeBasedPanel {
     public set backPlateMargin(value: number) {
         this._backPlateMargin = value;
 
-        this.children.forEach((control) => {
-            this._updateCurrentMinMax(control.position);
-        });
+        if (this._children.length >= 1) {
```
Comment (CraigFeldspar) [inline gui/src/3D/controls/touchHolographicMenu.ts:46]: nit: `if (this._children.length)`

**Review by CraigFeldspar — APPROVED** [review]
LGTM !

**Review by RaananW — APPROVED** [review]

File: gui/src/3D/controls/touchHolographicMenu.ts (hunk 1 above)
Comment (rickfromwork) [reply gui/src/3D/controls/touchHolographicMenu.ts:46]: Sorry, saw this too late!

Comment (sebavan) [reply gui/src/3D/controls/touchHolographicMenu.ts:46]: np I ll add it in the nightly

**Pull request merged**